atomica/defaults.py

```diff
@@ -76,16 +74,15 @@ def default_project(which=None, do_run=True, addprogs=True, verbose=False, show_
         ('hiv_dyn', 'HIV care cascade with demography (2 populations)'),
         ('tb_simple', 'Tuberculosis (1 population)'),
         ('tb_simple_dyn', 'Tuberculosis with demography (1 population)'),
+        ('tb', 'Tuberculosis with transmission dynamics'),
```

Comment:
Do we really need to duplicate this list between frameworks and projects? @robynstuart @RomeshA ?

Reply:
It's probably not essential, but note that generating this list would mean needing to open/read all of the framework files in the directory, and if we want to keep the population numbers in e.g. 'Tuberculosis (1 population)` then we might need to read in the databooks as well, so it could be a bit slower

Reply:
@RomeshA Oh no I just mean the (almost identical) list appears twice in `defaults.py`, it seems we could get away with one, especially since afaik the "default frameworks" list isn't actually used anywhere user-facing...

Reply:
Ah ok! Yes I don't think it needs to be duplicated. But it is used in the CAT in `rpcs.py` in `get_framework_options()` - so I think the framework one populates the dropdown on the Frameworks page? I think it should be fine to populate it with `default_project()` instead of `default_framework()` though

Reply:
OK cool, if it's OK I'll leave that with you then!

Reply:
OK let's merge this and do that in a future PR since it's not essential here